dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -83,7 +84,14 @@ def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions):
                             valuePresent[field][row[field]] = True
                     record = model(**row)
                 if not skipInsert:
-                    interface.session.merge(record)
+                    try:
+                        interface.session.merge(record)
+                    except IntegrityError as e:
+                        # Hit a duplicate value that violates index, skip this one
+                        print("".join(["Warning: Skipping this row: ",str(row)]))
```

Comment:
Nice, I like the soft landing here instead of just letting the db blow up if we get duplicates.